api/api.php

```diff
@@ -933,8 +933,20 @@
 
     switch ($app->request()->getMethod()) {
         case 'POST':
-            $newRecord = $entriesService->createEntry($table, $payload, $params);
-            $params[$tableGateway->primaryKeyFieldName] = $newRecord[$tableGateway->primaryKeyFieldName];
+            if ($table == 'directus_users') {
```

Comment:
Also this looks good.

Can you check https://github.com/directus/directus/blob/master/api/routes/A1/Entries.php#L23 and implement it as well for **API 1.1 for 6.4 on version/6.4 branch**